### DISCUSSION TITLE
Translation .po files

### DISCUSSION THREAD

**Discussion by arif-ali** [sosreport/sos#3775] [Q&A]
Another question for old contributors on the team.

With recent uploaded package of 4.8.0 into Debian, I saw a link that is automatically produced to show the current status of the translations. That showed majority of not being complete.

Further analysis showed me that these are well out if date, even to the locations that they are pointing to.

Ultimately, do we have plans to get these updated to point to the right locations, so that these could be more current.

A lot has changed in the project since the last time the translations were updated, so can see it would be potentially be a big job to align.

**Comment by jcastill** [comment]
It can be a big job, but doesn't need to be done all at once, I imagine. Also, the biggest work will be done in the first update, and then it should be a matter of just small incremental updates, right?
Could this be done with an Open Source translation AI? Not sure if there is something available in github, but it may make things easier and quicker, and we'll just need to review the results - again for the first big update at least.

**Reply by pmoravec** [reply]
I would vote for some automated translation using some tool (citation needed) as well, and optional manual corrections afterwards. That can save a lot of tedious manual work.

**Comment by jcastill** [comment]
Anyway if we decide to update the translations, I could do Spanish and some first touches at some romance languages, and do an call for volunteers in RH for the rest.
